lib/extensions/ar_adapter/ar_checkin_checkout_callbacks.rb

```diff
@@ -20,3 +20,73 @@ def log_after_checkout
     logger.debug { "#{self.class.name.demodulize}##{__method__}, #{connection_info_for_logging}" } if logger && ActiveRecord::Base.connected?
   end
 end
+
+
+
```

Comment:
Come one now @NickLaMuro, do you really want to add more lines than you need to.  :trollface:   😉 

Although this is just a POC to see if this helps the web socket worker issue.